app/ui/browser-australis/selectors/pages.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+// You might just want to use `getOrderedPageIds` instead.
```

Comment:
Doesn't warrant a warning I'd say, there's legitimate use cases for getting a pages array. Most times, however, it's not, hence the comment.